index.html

```diff
@@ -0,0 +1,1312 @@
+                </div>
+                <div class="dot"></div>
+                <div class="subtitle">
+                    <div class="subtitle-text">2015</div>
+                </div>
+            </div>
+        </div>
+        <div class="description">Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.</div>
+        <div class="skills-container">
+            <div class="skill">
+                <div class="skill-text">html</div>
+            </div>
+            <div class="skill">
+                <div class="skill-text">Ruby on rails</div>
+            </div>
+            <div class="skill">
+                <div class="skill-text">css</div>
+            </div>
+            <div class="skill">
+                <div class="skill-text">javaScript</div>
+            </div>
+        </div>
+        <div class="project-button-container">
+            <div class="project-button">
+                <div class="project-button-text">See project</div>
+            </div>
+           
+        </div>
+    </div>
+    <img class="image" src="img/work/port1.png" />
+</div>
+</section>
+<section class="Portfolio__About__MySelf"> 
+            <div class="portfolio__card__myself">
+            <h4 class="About__MySelf__header">About Myself</h4>
+            <div class="About__MySelf__Content">
+              <p>Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            </div>
+            <div class="About__MySelf__layout">
+              <div class="About__MySelf__design">
+                <h3 class="About__MySelf__Connect">LET'S CONNECT</h3>
+              </div>
+              <div class="social-media">
+              <ul>
+                <li><a href=""><img src="img/t_icon.png" alt="Twitter icon" /></a></li>
+                <li><a href=""><img src="img/Li_icon.png" alt="LinkedIn icon" /></a></li>
+                <li><a href=""><img src="img/m_icon.png" alt="Medium icon" /></a></li>
+                <li><a href=""><img src="img/_gth_icon.png" alt="Github icon" /></a></li>
+                <li><a href=""><img src="img/finger_c.png" alt="finger icon" /></a></li>
+            
+            </ul>
+           </div>
+           </div>
+           <div class="resume__tag">
+            <a class="resume__button"><p>Get my resume</p></a>
+           </div>
+           </div>
+       
+<!-- languages portfolio -->
+<div class="languages_container">
+    <div class="languages_heading-container">
+         
+        <div class="heading">Languages</div>
+        <div class="chevron-container">
+            <div class="chevron">
+                <img class="framework__arr" src="img/about_page/arr_d.svg">
+            </div>
+        </div>
+    </div>
```

Comment:
- [ ] Kindly ensure your design matches the given design for the `About Myself` section as well to improve the UI and match the given design.


Your design
![image](https://github.com/AsciencioAlex/desktop-portfolio/assets/81932478/a48aa96b-1588-43ae-99e8-9382a91709dc)



Given design
![image](https://github.com/AsciencioAlex/desktop-portfolio/assets/81932478/64bb27b8-1f82-4897-b960-322794fdbbfb)


- Align the languages section correctly. Add the missing arrow icons and align them as is in the given design as well.

Yours
![image](https://github.com/AsciencioAlex/desktop-portfolio/assets/81932478/ae5a563b-fb2c-4d15-b974-f9e4a5a94ae2)

Given
![image](https://github.com/AsciencioAlex/desktop-portfolio/assets/81932478/7a3389c2-e56a-4311-ae0c-5e330703138e)

- Add some margin between the `LET'S CONNECT` and the icons. Make sure your icons are aligned in a straight line.

Yours
![image](https://github.com/AsciencioAlex/desktop-portfolio/assets/81932478/9ad7f7a7-1bfb-46c9-ba0a-06c98986ee0f)

Given
![image](https://github.com/AsciencioAlex/desktop-portfolio/assets/81932478/7f4af1d5-2389-4160-ac20-e75d6e68d8d1)